flink-table/flink-table-common/src/main/java/org/apache/flink/table/plan/stats/ColumnStats.java

```diff
@@ -118,40 +81,10 @@ public Integer getMaxLen() {
         return maxLen;
     }
 
-    /**
-     * Deprecated because Number type max/min is not well supported comparable type, e.g. {@link
-     * java.util.Date}, {@link java.sql.Timestamp}.
-     *
-     * <p>Returns null if this instance is constructed by {@link ColumnStats.Builder}.
-     */
-    @Deprecated
-    public Number getMaxValue() {
-        return maxValue;
-    }
-
-    /**
```

Comment:
It looks like you have removed the javadoc associated with the getMax method

Reply:
Yes. Because the legacy construct `{@link ColumnStats#ColumnStats(Long, Long,
     * Double, Integer, Number, Number)}.` has been removed.